optimum/exporters/ipex/model_patcher.py

```diff
@@ -73,13 +93,10 @@ def _patch_llama_model(model):
         raise ImportError(
             f"Only transformers versions {_TRANSFORMERS_MIN_VERSION} ~ {_TRANSFORMERS_MAX_VERSION} are verified."
         )
-    convert_functions(model, LlamaModel, "forward", _llama_model_forward)
-    convert_functions(model, LlamaRMSNorm, "forward", _llama_layer_norm_forward)
-    convert_class(model, LlamaDecoderLayer, _IPEXLlamaDecoderLayer, model.config)
-    return model
-
-
-def _patch_model(model):
     if isinstance(model, LlamaForCausalLM):
         model = _patch_llama_model(model)
+    elif isinstance(model, BertForQuestionAnswering):
+        model = _patch_bert_model(model)
+    elif isinstance(model, ViTForImageClassification):
+        model = _patch_vit_model(model)
```

Comment:
why only patching these specific tasks ? the conversion should be appliable to any `model.config.model_type==bert` (or vit)

Reply:
You were right, I am changing on it :) Thx!

Reply:
fixed